framework/python/src/test_orc/test_orchestrator.py

```diff
@@ -148,6 +174,19 @@ def run_test_modules(self):
     self._test_in_progress = False
     self.get_session().set_report_url(report.get_report_url())
 
+    # Set testing description
+    test_pack: TestPack = self.get_test_pack(device.test_pack)
+
+    # Default message is empty (better than an error message).
+    # This should never be shown
+    message: str = ""
+    if report.get_status() == TestrunStatus.COMPLIANT:
+      message = test_pack.get_message("compliant_description")
+    elif report.get_status() == TestrunStatus.NON_COMPLIANT:
+      message = test_pack.get_message("non_compliant_description")

```

Comment:
What is the purpose of these messages?  I don't seem to see their usage anywhere.

Reply:
Those messages are going to be used on the new Testrun completion modal, I will send the design over